### DISCUSSION TITLE
WPF UI extension  are an outdated version of the VS market

### DISCUSSION THREAD

**Discussion by weqasds** [lepoco/wpfui#1294] [Q&A]
As shown in Fig
![image](https://github.com/user-attachments/assets/6ef8593b-86da-45b0-834d-1d0243b61666)
But I found out in the source code that the version number of this extension has reached 3.05, do I need to build this extension and install it myself?

**Comment by roface** [comment]
Even if the version has updated to 4.0.0, VS market still shows 3.0.4.
It was really hard for new users to apply 4.0.0.

**Reply by Djohnnie** [reply]
I've submitted a PR #1344 that updates the VS extension to use the latest 4.0.0 version. Let's hope to get it out there :)
